core/tests/protractor_utils/editor.js

```diff
@@ -1349,6 +1349,42 @@ var _runFromFeedbackTab = function(callbackFunction) {
   return result;
 };
 
+var getSuggestionThreads = function() {
+  return _runFromFeedbackTab(function() {
+    var suggestionRowClassName = '.protractor-test-oppia-feedback-tab-row';
```

Comment:
Nit: make this a constant. SUGGESTION_ROW_CLASS_NAME

Ditto in a few places below.